Project.toml

```diff
@@ -5,6 +5,8 @@ version = "0.1.0"
 
 [deps]
 BundleMethod = "e8a6717a-069f-43d5-9ac2-a49b1ec5be7b"
+Debugger = "31a5f54b-26ea-5ae9-a837-f05ce5417438"
```

Comment:
Please remove the dependency on packages `Debugger` and `GLPK`

Reply:
Done!